Makefile

```diff
@@ -789,3 +793,37 @@ update-machine-types: #Update machine_types.go
 	go build -o hack/machine_types/machine_types  ${KOPS_ROOT}/hack/machine_types/
 	hack/machine_types/machine_types --out upup/pkg/fi/cloudup/awsup/machine_types.go
 	go fmt upup/pkg/fi/cloudup/awsup/machine_types.go
+
+#-----------------------------------------------------------
+# development targets
+
+# dev-upload-nodeup uploads nodeup to GCS
+.PHONY: dev-upload-nodeup
+dev-upload-nodeup: bazel-crossbuild-nodeup
+	mkdir -p ${BAZELUPLOAD}/kops/${VERSION}/linux/amd64/
+	cp -fp bazel-bin/cmd/nodeup/linux_amd64_pure_stripped/nodeup ${BAZELUPLOAD}/kops/${VERSION}/linux/amd64/nodeup
+	(${SHASUMCMD} ${BAZELUPLOAD}/kops/${VERSION}/linux/amd64/nodeup | cut -d' ' -f1) > ${BAZELUPLOAD}/kops/${VERSION}/linux/amd64/nodeup.sha1
+	${UPLOAD} ${BAZELUPLOAD}/ ${UPLOAD_DEST}
+
+# dev-upload-protokube uploads protokube to GCS
+.PHONY: dev-upload-protokube
+dev-upload-protokube: bazel-protokube-export # Upload kops to GCS
+	mkdir -p ${BAZELUPLOAD}/kops/${VERSION}/images/
+	cp -fp ${BAZELIMAGES}/protokube.tar.gz ${BAZELUPLOAD}/kops/${VERSION}/images/protokube.tar.gz
+	cp -fp ${BAZELIMAGES}/protokube.tar.gz.sha1 ${BAZELUPLOAD}/kops/${VERSION}/images/protokube.tar.gz.sha1
+	${UPLOAD} ${BAZELUPLOAD}/ ${UPLOAD_DEST}
+
+# dev-copy-utils copies utils from a recent release
+# We don't currently have a bazel build for them, and the build is pretty slow, but they change rarely.
+.PHONE: dev-copy-utils
+dev-copy-utils:
+	mkdir -p ${BAZELUPLOAD}/kops/${VERSION}/linux/amd64/
+	cd ${BAZELUPLOAD}/kops/${VERSION}/linux/amd64/; wget -N https://kubeupv2.s3.amazonaws.com/kops/1.11.0-alpha.1/linux/amd64/utils.tar.gz
```

Comment:
Should the version here be hard coded or do we want to change it to `${VERSION}`

Reply:
So the gotcha here is that (at the moment) the utils bundle isn't built using bazel.  So I'm copying it from a recent version.  I agree it is very far from ideal!